src/pages/en/guides/markdown-content.md

```diff
@@ -611,16 +644,27 @@ export default defineConfig({
 });
 ```
 
-We also suggest [diving into their theme documentation](https://github.com/shikijs/shiki/blob/main/docs/themes.md#loading-theme) to explore more about themes, light vs dark mode toggles, or styling via CSS variables.
+We also suggest reading [Shiki's own theme documentation](https://github.com/shikijs/shiki/blob/main/docs/themes.md#loading-theme) to explore more about themes, light vs dark mode toggles, or styling via CSS variables.
 
 #### Prism configuration
 
-When using Prism, you'll need to add a stylesheet to your project for syntax highlighting. If you're just getting started and prefer to use Prism over Shiki, we suggest:
+If you'd like to switch to `'prism'` by default, or disable syntax highlighting entirely, you can use the `markdown` config object:
```

Comment:
We mention you can disable highlighting here but don’t show how.

Reply:
took a pass at this in a larger edit